src/internal/globalVars.ts

```diff
@@ -41,4 +42,8 @@ export class GlobalVars {
   public static appButtonHoverLeaveHandler: () => void;
   public static meetingRoomCapabilitiesUpdateHandler: (capabilities: meetingRoom.MeetingRoomCapability) => void;
   public static meetingRoomStatesUpdateHandler: (states: meetingRoom.MeetingRoomState) => void;
+  public static capableParticipantsChangeHandler: (participantChange: remoteCamera.CapableParticipants) => void;
```

Comment:
These function looks more generic since they are not under namespace, can we prefix with remoteCamera for these functions?